packages/localize-tools/src/modes/transform.ts

```diff
@@ -218,17 +221,17 @@ class Transformer {
           '_LIT_LOCALIZE_CONFIGURE_TRANSFORM_LOCALIZATION_'
         )
       ) {
-        return ts.createObjectLiteral(
+        return factory.createObjectLiteralExpression(
```

Comment:
Did this API exist on typescript@4.7? Checking if this is a breaking change for localize-tools.

Reply:
If it is, we probably need https://github.com/lit/lit/pull/3800, but with `~5.0.0` cc @aomarks

Reply:
Also, if it *is* compatible, then we need to add `~5.0.0` (see https://github.com/lit/lit/pull/3800)

Reply:
There's breaking changes in the TS APIs, so will do this as a breaking change to localize-tools and ts-transformers